Essential URL normalization cannot be disabled (#14019)
cloudflare/cloudflare-docs: content/rules/normalization/_index.md

When you enable URL normalization, all incoming URLs are normalized before they pass to subsequent global network features that accept a URL input, such as WAF custom rules, Workers, and Access. Rule expressions that filter traffic based on URLs will therefore trigger correctly, regardless of the format of the incoming URL. When URL normalization is disabled, Cloudflare forwards the URL to origin in its original form.

{{<Aside type="warning">}}

> When traffic is proxied via Cloudflare, essential request URL normalization is always applied regardless whether URL normalization is enabled for a specific zone. For instance, compression of two or more adjacent slashes in a request URL into a single slash cannot be disabled by turning URL normalization off.

Reviewer: ```suggestion
When traffic is proxied via Cloudflare, essential request URL normalization is always applied regardless whether URL normalization is enabled for a specific zone. For example, you cannot disable the conversion of two or more adjacent slashes into a single slash in a request URL by turning off URL normalization.
```
